boot/bootutil/include/bootutil/bootutil.h

```diff
@@ -83,7 +83,13 @@ fih_ret boot_go_for_image_id(struct boot_rsp *rsp, uint32_t image_id);
 
 struct boot_loader_state;
 void boot_state_clear(struct boot_loader_state *state);
-fih_ret context_boot_go(struct boot_loader_state *state, struct boot_rsp *rsp);
+
+fih_int context_boot_go_flash(struct boot_loader_state *state, struct boot_rsp *rsp);
```

Comment:
`fih_int` was replaced by `fih_ret` in this change:
https://github.com/mcu-tools/mcuboot/commit/5047f032c93fd8957be6529f8f1ad7a3fe626057